src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/FtaCommunicationFields.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FtaCommunicationFields {
+
```

Comment:
for requirements 5/6, add in
```
private String ftaCommunicationFilter;
private String tribunalCommunicationFilter;
```

Reply:
(make enums for them as they are fixed Strings)

Reply:
Done